collector/filesystem_freebsd.go

```diff
@@ -108,17 +95,40 @@ func (c *filesystemCollector) Update(ch chan<- prometheus.Metric) (err error) {
 			log.Debugf("Ignoring mount point: %s", name)
 			continue
 		}
-		c.size.WithLabelValues(name).Set(float64(mnt[i].f_blocks) * float64(mnt[i].f_bsize))
-		c.free.WithLabelValues(name).Set(float64(mnt[i].f_bfree) * float64(mnt[i].f_bsize))
-		c.avail.WithLabelValues(name).Set(float64(mnt[i].f_bavail) * float64(mnt[i].f_bsize))
-		c.files.WithLabelValues(name).Set(float64(mnt[i].f_files))
-		c.filesFree.WithLabelValues(name).Set(float64(mnt[i].f_ffree))
-	}
+		ch <- prometheus.MustNewConstMetric(
+			sizeDesc,
+			prometheus.GaugeValue,
+			float64(mnt[i].f_blocks) * float64(mnt[i].f_bsize),
+			mpd.device, mpd.mountPoint, mpd.fsType,
```

Comment:
mpd is undefiend. (I guess this wouldn't compile on freebsd. But on linux, this file is not touched, so the error isn't detected.)